src/ui/3d/map3dexportwidget.ui

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>Map3DExportWidget</class>
+ <widget class="QWidget" name="Map3DExportWidget">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>400</width>
+    <height>300</height>
+   </rect>
+  </property>
+  <layout class="QGridLayout" name="gridLayout_2">
+   <item row="0" column="0">
+    <layout class="QGridLayout" name="gridLayout">
+     <item row="1" column="1">
+      <widget class="QLineEdit" name="folderPathLineEdit"/>
+     </item>
+     <item row="0" column="0">
+      <widget class="QLabel" name="sceneNameLabel">
+       <property name="text">
+        <string>Scene name</string>
+       </property>
+      </widget>
+     </item>
+     <item row="1" column="0">
+      <widget class="QLabel" name="folderLAbel">
+       <property name="text">
+        <string>Folder</string>
+       </property>
+      </widget>
+     </item>
+     <item row="0" column="1" colspan="2">
+      <widget class="QLineEdit" name="sceneNameLineEdit"/>
```

Comment:
You should use QgsFileWidget here, that's got lots of built in logic for selection of folders